HW03 Brock.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "COVID Mapping"
+author: "Brock Adams"
+date: "7/20/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(readr) #reading rectangular text data
+library(ggplot2) #plotting data
+library(magrittr) #piping
+```
+
+```{r COVID Data import}
+COVID_url = "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"
+COVID_data <- read_csv(url(COVID_url))
+COVID_CA <- subset(COVID_data, state == "California")
```

Comment:
Oh I like the use of this subset function! Is this different from the select function because it lets you choose rows and columns?

Reply:
Yeah, that's right! The subset function let's you identify specific values, not only variables, like the select function.